.mockery.yaml

```
@@ -8,4 +8,5 @@ template: testify
 packages:
   github.com/nice-pea/npchat/internal/controller/http2/middleware: 
   github.com/nice-pea/npchat/internal/controller/http2/register_handler: 
-  github.com/nice-pea/npchat/internal/usecases/events: 
\ No newline at end of file
+  github.com/nice-pea/npchat/internal/usecases/events: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)